ui/src/Pages/AlbumPage/AlbumPage.js

```diff
@@ -124,6 +125,17 @@ function AlbumPage({ match }) {
     [setOnlyFavorites, refetch]
   )
 
+  useEffect(() => {
+    LazyLoad.loadImages(document.querySelectorAll('img[data-src]'))
+    return () => LazyLoad.disconnect()
+  })
```

Comment:
Add no dependencies to useEffect to only call disconnect when unloaded.
disconnect was called on each page load without this

```suggestion
  }, [])
```

Reply:
Yeah, I know this, but I forget it accidentally.
Thank you for noticing it!